deployments/gke-job/cron-job.yaml

```diff
@@ -0,0 +1,47 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: batch/v1beta1
```

Comment:
Does this work with /v1 version?

Reply:
Thanks for noticing this, some thoughts below:

`CronJob` kind is supported through `batch/v1beta1` for version kubernetes api `v1.20` 
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#cronjob-v1beta1-batch.

However, the latest kunernetes api v1.22 does support CronJob through `batch/v1`.

I have installed my kubectl through CloudSDK, updating it through CloudSDK `gcloud components update` got me to `v1.20`. So, in conclusion, I think it might be good to support what gcloud is supporting.

Reply:
Ack. Thanks for the explanation!